common/src/main/scala/uk/ac/wellcome/models/Work.scala

```diff
@@ -4,26 +4,57 @@ import com.sksamuel.elastic4s.Indexable
 import uk.ac.wellcome.utils.JsonUtil._
 
 /** A representation of a work in our ontology */
-case class Work(title: Option[String],
+trait Work extends Versioned with Identifiable{
+  val title: Option[String]
+val  sourceIdentifier: SourceIdentifier
+val                version: Int
```

Comment:
Wat is going on with this whitespace?

Reply:
For some reason travis is not formatting...